dataactbroker/fileRoutes.py

```diff
@@ -136,6 +136,11 @@ def complete_generation(generation_id):
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
         return file_manager.complete_generation(generation_id)
 
+    @app.route("/v1/upload_detached_file/", methods=["POST"])
+    def upload_detached_file():
```

Comment:
I was debating updating it to use them but I never quite did it. Do you think we should do that or should that be for the next ticket?